src/store/actions/index.js

```diff
@@ -6,22 +6,22 @@ export const LOGIN_START = "LOGIN_START";
 export const LOGIN_SUCCESS = "LOGIN_SUCCESS";
 export const LOGIN_ERROR = "LOGIN_ERROR";
 
-export const login = (creds, history) => (dispatch) => {
+export const login = (creds, history) => dispatch => {
   dispatch({ type: LOGIN_START });
   axiosWithAuth()
     .post(`/auth/login`, creds)
-    .then((res) => {
+    .then(res => {
       console.log(res);
       setTimeout(() => {
         localStorage.setItem("token", res.data.token);
         localStorage.setItem("user_id", res.data.signIn.id);
         setTimeout(() => {
           dispatch({ type: LOGIN_SUCCESS });
-          history.push("/videos");
+          history.push(`/users/${res.data.signIn.username}`);
```

Comment:
Need to `setItem` for `username` so we can pull it elsewhere so that 'profile' works correctly. However, we'll want to use `useParams` for other users profiles you want to visit.